ui/app/src/App.tsx

```diff
@@ -116,9 +115,6 @@ export const App: React.FunctionComponent = () => {
                         </Routes>
                       </GenericErrorBoundary>
                     </Stack.Item>
-                    <Stack.Item grow>
```

Comment:
what's the reason for removing the footer? it's likely this could contain generic terms of use / privacy / support type links in future